use_cases.md

```diff
@@ -19,51 +18,50 @@ The new CaaS service from Acme Corp. will include:
   * Shared Public IP addresses.
   * Shared Persistent Storage.
   * Automatic backup of volumes.
-  * Shared routing layer with shared wildcard certificate.
-  * Multiple Namespaces isolation.
+  * Shared routing layer with a shared wildcard certificate.
+  * Multiple Namespaces isolations.
   * Single user account.
   * Resources Quotas and Limits.
   * Self Service Provisioning portal.
   * Shared Application Catalog.
 
 - **Private CaaS**:
 
+
   * Dedicated infra and worker nodes.
   * Dedicated registry.
   * Dedicated routing layer with dedicated wildcard certificates.
   * Dedicated Public IP addresses.
   * Dedicated Persistent Storage.
   * Automatic backup of volumes.
   * Shared control plane.
-  * Multiple Namespaces isolation.
+  * Multiple Namespaces isolations.
   * Resources Quotas and Limits.
   * Self Service Provisioning portal.
   * Dedicated Application Catalog.
   * Multiple user accounts.
-  * Optional access to VMs, Storage, Networks, DBaaS, and other managed
-    traditional services from the IaaS offer.
+  Optional access to VMs, Storage, Networks, DBaaS, and other managed traditional services from the IaaS offering.
```

Comment:
Why put this line out of the list? Please, ensure 80 chars as max column length.